mycroft/skills/intent_services/adapt_service.py

```diff
@@ -196,7 +196,7 @@ def take_best(intent, utt):
                 # TODO - Shouldn't Adapt do this?
                 best_intent['utterance'] = utt
 
-        for idx, utt in enumerate(raw_utt):
+        for idx, utt in enumerate(combined):
```

Comment:
Yes it would, I don't think you'd see a noticeable difference though.

Parsing both raw and normalized was added way back when to make things easier for skill creators. In case they enter a keyword like "that's" or something that gets normalized to "that is".

Technically the normalization could easily be done when adding the keyword into adapt instead.

Reply:
Cool, just curious, will keep it in mind for later